playground/scripts/render-policy.sh

```diff
@@ -11,6 +11,13 @@ api_key=${7:-}
 endpoint=${8:-}
 agent_group=${9:-default}
 action=${10:-apply}
+skipverify=${11:-false}
+
+if [[ "$skipverify" == "true" ]]; then
```

Comment:
Please add {} around the variable name like we follow for all the other variables